### ISSUE TITLE
Getting "ReferenceError: Can't find variable: process"...

### ISSUE THREAD

**Issue by digitalmoksha** [wooorm/remark#203]
I've started incorporating remark 6.0.0 into an html file, and i'm running into the error

```
ReferenceError: Can't find variable: process
  VFile — remark.js:15508
  VFile — remark.js:15502
  process — remark.js:15114
  eval code
  eval
  _evaluateOn
  _evaluateAndWrap
  evaluate
```

which resolves down to the line `this.cwd = process.cwd();`

This is with Safari 9.1.2

I'm initially using like this:

```
        var processor = remark().use(remarkHTML);
        var doc = processor.parse(text);
```

**Comment by wooorm** [comment]
I think this is a problem with your build process, not with remark itself.

How are you building remark?

**Comment by digitalmoksha** [comment]
Actually i'm using the latest release file, remark.js.  I didn't build it.

**Comment by digitalmoksha** [comment]
```
    <script src="remark.js"></script>
    <script src="remark-html.js"></script>
```

**Comment by wooorm** [comment]
Thanks for reporting. I’ve fixed this in 414e775, and updated the [sources for 6.0.0](https://github.com/wooorm/remark/releases/tag/6.0.0) accordingly.

I \*really\* suggest you use browserify yourself though!

**Comment by digitalmoksha** [comment]
Cool, thank you, that fixed it.

I did start trying to build them, but ran into a couple stumbling.  Then I realized you had already browserfied them, as `remark.js` and `remark.min.js`.  It's better not to use those?

Appreciate you fixing it so quickly!
